### PR TITLE
Minor correction of a few punctuational errors and CSS suggestion

### PR DESCRIPTION
**Pull request by zziccardi**

### CODE REVIEW

**Comment by brntbeer** [comment]
hey @zziccardi, would you mind taking some before and after shots of these changes being rendered locally?

**Comment by zziccardi** [comment]
@brntbeer No problem!

For 53ac894, see https://www.dropbox.com/s/l9slnjbs7fu0zyu/%23branding%20before.png then https://www.dropbox.com/s/cjhr63sw6vg4do8/%23branding%20after.png.

For 730f9fd, see https://www.dropbox.com/s/2m5xj8lijkyi10n/hyphen-minus%20symbols%20%28before%29.png then https://www.dropbox.com/s/pdrz9s1swwl2f00/em%20dashes%20%28after%29.png.

All screenshots were taken with Chrome 35.0.1916.114 beta on OS X 10.9.3. I also tested in Safari 7.0.3 and Firefox 30 beta and everything rendered identically.

**Comment by iMakeitRain** [comment]
IM LOVING THIS BUT FEEL LOST AN HOPE THAT IM ON THE RIGHT TRACK

File: branching/index.html [hunk 1]
```diff
@@ -633,7 +633,7 @@ <h4 id="merge-conflicts">
     out what is happening on each.</p>
 
     <p>First we'll create a new branch to add the Erlang programming language
-    Hello World example - we want to do this in a branch so that we don't
```
Comment (iMakeitRain) [inline branching/index.html:636]: OK 
I SHOULD OF SPENT MORE TIME WITH WORDPRESS LOO

Comment (iMakeitRain) [inline branching/index.html:636]: LOL

**Pull request closed without merging**